src/NodeRTLib/CppTemplates/Event.cpp

```diff
@@ -70,16 +70,16 @@
 
       if (info.Length() < 2 || !info[0]->IsString() || !info[1]->IsFunction())
       {
-        Nan::ThrowError(Nan::Error(NodeRT::Utils::NewString(L"wrong arguments, expected a string and a callback")));
+        Nan::ThrowError(Nan::Error(NodeRT::Utils::NewString("wrong arguments, expected a string and a callback")));
         return;
       }
 
       String::Value eventName(info[0]);
```

Comment:
Since String::Value returns a unicode string, we actually compare a unicode string to non-unicode here and will probably see some issues. Instead, we can either use String::Utf8Value which returns char*, or keep the previous version which compares 2 unicode strings.